app/src/main/java/com/babylon/wallet/android/presentation/settings/securitycenter/SecurityCenterScreen.kt

```diff
@@ -16,34 +16,57 @@ import androidx.compose.material3.Icon
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.babylon.wallet.android.R
 import com.babylon.wallet.android.designsystem.theme.RadixTheme
 import com.babylon.wallet.android.designsystem.theme.RadixWalletTheme
+import com.babylon.wallet.android.domain.usecases.SecurityPromptType
 import com.babylon.wallet.android.presentation.ui.composables.DSR
 import com.babylon.wallet.android.presentation.ui.composables.RadixCenteredTopAppBar
+import rdx.works.profile.data.model.BackupState
 
 @Composable
 fun SecurityCenterScreen(
     modifier: Modifier = Modifier,
+    securityCenterViewModel: SecurityCenterViewModel = hiltViewModel(),
     onBackClick: () -> Unit,
     onSecurityFactorsClick: () -> Unit,
+    onBackupConfigurationClick: () -> Unit,
 ) {
-    SecurityCenterContent(modifier, onBackClick, onSecurityFactorsClick)
+    val state by securityCenterViewModel.state.collectAsStateWithLifecycle()
```

Comment:
ℹ️  just a reminder that here the  `collectAsStateWithLifecycle` is behaving as `collectAsState` since you already collecting the state in the `SecurityCenterViewModel`